_includes/toolkit/phone-assessment-interviews.html

```diff
@@ -7,6 +7,30 @@
     <a href="{{ site.baseurl }}/toolkit/assessment-strategy/phone-interview-template.docx" target="_blank" class="chp-toolkit__link">{% include svg/file-word.svg %}Phone Interview Template
     </a>
   </li>
+  <li class="chp-toolkit__item">
+    <a href="{{site.baseurl}}/toolkit/phone-assessment-interviews/email-template-sme-availability/" class="chp-toolkit__link">{% include svg/file-email.svg %}Email to SMEs for Interview Availability
+    </a>
+  </li>
+  <li class="chp-toolkit__item">
+    <a href="{{site.baseurl}}/toolkit/phone-assessment-interviews/example-sme-availability-worksheet/" class="chp-toolkit__link">{% include svg/file-excel.svg %}SME Interview Availability Worksheet
+    </a>
+  </li>
+  <li class="chp-toolkit__item">
+    <a href="{{site.baseurl}}/toolkit/phone-assessment-interviews/sme-recusal-email-template/" class="chp-toolkit__link">{% include svg/file-email.svg %}SME Recusal Email Template
```

Comment:
> `{% include svg/file-email.svg %}`

@humancompanion-usds: Here's one of the four spots where I'm referencing a new email template SVG that doesn't exist yet. I'm not sure where you got the other ones.

Reply:
I have a local copy of font awesome. I will pull this down and update.

Reply:
Thanks! ✨ ✉️